cr.sh

```diff
@@ -198,15 +198,7 @@ lookup_changed_charts() {
 
     local changed_files
     changed_files=$(git diff --find-renames --name-only "$commit" -- "$charts_dir")
-
-    local fields
-    if [[ "$charts_dir" == '.' ]]; then
-        fields='1'
-    else
-        fields='1,2'
-    fi
-
-    cut -d '/' -f "$fields" <<< "$changed_files" | uniq | filter_charts
+    find "$charts_dir" -maxdepth 1 -type d -not -path "$charts_dir" |filter_charts
```

Comment:
I don't think this will work as expected as it does not take into consideration the `changed_files`

Reply:
Thanks for pointing that out. I forgot to do the comparison against that var here; not sure how I missed that.